nix/status-go/build-status-go.nix

```diff
@@ -19,7 +19,7 @@ let
   args = removeAttrs args' [ "buildMessage" ]; # Remove our arguments from args before passing them on to buildGoPackage
   buildStatusGo = buildGoPackage (args // {
     pname = repo;
-    version = "${version}-${strings.substring 0 7 rev}-${host}";
+    version = "${rawVersion}-${strings.substring 0 7 rev}-${host}";
```

Comment:
I think you need to use the sanitized version here @jakubgs otherwise Nix will fail when creating the drv file.

Reply:
right, fixed